src/hotspot/share/gc/z/zMark.cpp

```diff
@@ -582,8 +582,9 @@ class ZMarkConcurrentRootsIteratorClosure : public ZRootsIteratorClosure {
     ZThreadLocalAllocBuffer::publish_statistics();
   }
 
-  virtual bool should_disarm_nmethods() const {
-    return true;
+  virtual ZNMethodEntry nmethod_entry() const override {
```

Comment:
According to the stye-guide we shouldn't use `override` yet. It's used in a few more places in this patch.

Reply:
Will remove.

Reply:
I wonder if there's a ticket for revising the style guide to include `override`, which does make the overriding more explicit.

Reply:
https://bugs.openjdk.java.net/browse/JDK-8254050

Reply:
I see; thank you.